lib/watir/wait.rb

```diff
@@ -246,5 +246,23 @@ def wait_while_present(timeout = nil)
       # it's not present
     end
 
+    #
+    # Waits until the element is stale.
+    #
+    # @example
+    #   element = browser.text_field(name: "abrakadbra").wait_until_stale
+    #
+    # @param [Fixnum] timeout seconds to wait before timing out
+    #
+    # @see Watir::Wait
+    # @see Watir::Element#stale?
+    #
+
+    def wait_until_stale(timeout = nil)
```

Comment:
1. What's the use case?
2. Why don't you provide opposite method `#wait_while_stale`?